tools/golang-json-rpc-tests/main.go

```diff
@@ -0,0 +1,412 @@
+package main
```

Comment:
Missing license header

Reply:
It was missing, so I’ve added “Licensed under the Apache License,” similar to the other examples.